lib/manageiq/providers/workflows/builtin_runner.rb

```diff
@@ -27,13 +27,10 @@ def cleanup(runner_context)
           BuiltinMethods.send("#{method}_cleanup", runner_context)
         end
 
-        def wait(timeout: nil, events: %i[create update delete])
-          # TODO: wait_for_task?
-        end
-
         def status!(runner_context)
           method = runner_context["method"]
           raise ArgumentError if method.nil?
+          return if !runner_context["running"]
```

Comment:
I'm not sure I understand this bit - Is this to prevent status being called twice?

Reply:
yes. status! is called 2-3 times once a task is complete

also, if a task fails in the original run, then we don't want it calling status

Reply:
> also, if a task fails in the original run, then we don't want it calling status

In the container_runner classes we return if an error has already been set in the runner context, would that work for this case?

Reply:
The check on Error was the inspiration.
The thinking was that if we set the `output` / `running=false`, then we were done with this task and no reason to run status to check further